backend/app/api/endpoints/groups.py

```
@@ -208,6 +223,221 @@ def list_members(
+
+    try:
+        db.commit()
+    except IntegrityError:
+        db.rollback()
+        raise HTTPException(
+            status_code=status.HTTP_409_CONFLICT,
+            detail=(
+                f"Entity member ({member_create.entity_type}, {member_create.entity_id}) "
+                "already exists in this group"
+            ),
+        )
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)